airbyte-cdk/bulk/core/load/src/test/kotlin/io/airbyte/cdk/command/MockCatalogFactory.kt

```diff
@@ -14,9 +14,9 @@ import io.micronaut.context.annotation.Requires
 import jakarta.inject.Named
 import jakarta.inject.Singleton
 
-@Factory
-@Replaces(factory = DestinationCatalogFactory::class)
-@Requires(env = ["test"])
+//@Factory
+//@Replaces(factory = DestinationCatalogFactory::class)
+//@Requires(env = ["test"])
```

Comment:
@johnny-schmidt I did this yesterday as a quick hack just to get the test to run (otherwise micronaut threw a duplicate bean error). I think the easiest solution is to add another env to the Requires clause? `@Requires(env=["test", "mock_catalog_factory"])` and then add that new env to all the tests that rely on this bean?

Reply:
I could also add some Requires thing to the MockDestinationWrite, but then I'd need to plumb it down to the CliRunner

(... I could _also_ move this stuff to `testIntegration` and wire up a proper integration test task :shrug: no strong opinion. But I do think it would be nice for the MockCatalogFactory to be an explicit opt-in)